tests/e2e/must-gather_suite_test.go

```diff
@@ -23,11 +23,18 @@ var _ = Describe("Backup and restore tests with must-gather", func() {
 	}
 
 	var _ = AfterEach(func(ctx SpecContext) {
+		knownFlake = false
```

Comment:
NO we had this issue in the past of duplicating code from backup/restore tests to must-gather tests. Add code to `tearDownBackupAndRestore` and `runBackupAndRestore` functions

Reply:
Fixed.

Reply:
still has code outside `runBackupAndRestore` function

Reply:
Which ?

The following one? I don't like the following to be outside of DescribeTable so it's a first thing it's happening

```
			if CurrentSpecReport().NumAttempts > 1 && !knownFlake {
				Fail("No known FLAKE found in a previous run, marking test as failed.")
			}

```